src/dotnet/CarbonAware.Aggregator/src/CarbonAware/CarbonAwareAggregator.cs

```diff
@@ -0,0 +1,26 @@
+using CarbonAware.Model;
+using CarbonAware.Plugin;
+using Microsoft.Extensions.Logging;
+using System.Collections;
+
+namespace CarbonAware.Aggregator.CarbonAware
```

Comment:
Aggregator should be pluralized to match the format of other projects:  Aggregators